contracts/core/modules/follow/ApprovalFollowModule.sol

```diff
@@ -102,7 +111,7 @@ contract ApprovalFollowModule is IFollowModule, FollowValidatorFollowModuleBase
      * @param profileId The token ID of the profile to query approval with.
      * @param toCheck The address to query approval for.
      *
-     * @return
+     * @return bool Whether the address is approved or not.
```

Comment:
When returning a bool, we normally write `true if the address is approved and false otherwise` or similar, clearly outlining what a true and false return value mean.

Reply:
Resolved  in commit 2c30269f1a7b1fac30cae6934ed23d692e54ebb6.